INSTALL.md

```diff
@@ -52,22 +74,51 @@ To skip the secrets taks, just pass:
 
 The secret keys that have to be defined are:
 
-* backend_url
-* base_url
-* backend_token
-* secret_key
-* info_email
-* ssl_stapling_resolver
-* google_analytics
+* backend_url: The URL for requesting the backend. Ex: http://kernelci-backend/
+* base_url: The URL for requesting the frontend. Ex: http://kernelci-frontend/
+* backend_token: A GET+POST token. (See Manual tasks below for generating this token)
+* secret_key:  should be set to a random string, it is used internally by Flask.
```

Comment:
Hmmm... this might be confusing though.
All the above keys are required only for the frontend, and Flask is frontend only.

Now I see that having the same install file for both might be more confusing.
I would say that these should go into the kernelci-frontend-config install instructions rather than being here.

We can always link the two install files, and in the frontend one there should be written that it is necessary to install the backend first. I guess it makes more sense to split out the specific parts and keep the two separate (although they should have the same introductions etc...).

Reply:
I have splited changes on both github